test/iron-menubar-behavior.html

```diff
@@ -91,12 +103,55 @@
           menubar.selected = 0;
           menubar.items[0].click();
           setTimeout(function() {
-            assert.equal(document.activeElement, menubar.items[0], 'document.activeElement is last activated item');
+            assert.equal(Polymer.dom(document).activeElement, menubar.items[0], 'document.activeElement is last activated item');
             done();
           // wait for async in _onFocus
           }, 200);
         });
 
+        suite('left / right keys are reversed when the menubar has RTL directionality', function() {
+          var LEFT = 37;
+          var RIGHT = 39;
+
+          test('left key moves to the next item', function(done) {
+            var rtlContainer = fixture('rtl');
+            var menubar = rtlContainer.querySelector('test-menubar');
+            menubar.selected = 0;
+            menubar.items[1].click();
+            setTimeout(function() {
+              MockInteractions.pressAndReleaseKeyOn(menubar, LEFT);
+              setTimeout(function() {
+                assert.equal(Polymer.dom(document).activeElement, menubar.items[2],
+                    '`document.activeElement` should be the next item.');
+                assert.equal(menubar.selected, 1,
+                    '`menubar.selected` should not change.');
+                done();
+              // wait for async in _onFocus
+              }, 200);
```

Comment:
([This comment](https://github.com/PolymerElements/iron-menu-behavior/pull/41#discussion_r47314458) is still applicable.)